src/main/java/ui/components/NavigableListView.java

```diff
@@ -158,11 +158,11 @@ protected void setupKeyEvents() {
             }
             if (FIRST_ISSUE.match(e)) {
                 e.consume();
-                selectFirstItem();
+                selectnthItem(1);
```

Comment:
I think this should be named 'selectNthItem', to be more consistent with how we capitalise in names. Same for the other occurrences.